### PR TITLE
Django test helpers

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: germanium/django/utils.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                connection.creation.create_test_db(
+                    verbosity=verbosity,
+                    autoclobber=not interactive,
+                    keepdb=keepdb,
+                    serialize=connection.settings_dict['TEST'].get('SERIALIZE', True),
+                )
+
+                test_fixtures = getattr(settings, 'GERMANIUM_GLOBAL_FIXTURES', {}).get(alias, None)
+                if test_fixtures:
+                    call_command('loaddata', *test_fixtures, **{'verbosity': 0, 'database': alias})
```
Comment (zamazaljiri) [inline germanium/django/utils.py:31]: @rubickcz Ať nemusíš dělat diff a hledat změnu, tak je změna jen zde :)

**Review by zamazaljiri — APPROVED** [review]

File: germanium/django/utils.py (hunk 1 above)
Comment (matllubos) [reply germanium/django/utils.py:31]: sorry for that. Yes unfortunatelly I must copy whole code :(

Comment (rubickcz) [reply germanium/django/utils.py:31]: @zamazaljiri thanks
@matllubos why?

Comment (rubickcz) [reply germanium/django/utils.py:31]: Maybe this is some django function you want to alter...

File: germanium/django/runner.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+class GermaniumRunnerMixin:
+
+
+    def __init__(self, **kwargs):
+        self.refreshdb = kwargs.pop('refreshdb', False)
+        super().__init__(**kwargs)
+
+    def setup_databases(self, **kwargs):
+        return setup_databases(
+            self.verbosity, self.interactive, self.keepdb and not self.refreshdb, self.debug_sql,
```
Comment (rubickcz) [inline germanium/django/runner.py:15]: This will not work if I want to delete existing DB, create new, but keep it, or?

**Review by rubickcz — APPROVED** [review]

File: germanium/django/utils.py (hunk 1 above)
Comment (matllubos) [reply germanium/django/utils.py:31]: yes exactly

**Pull request merged**